.gitlab-ci.yml

```diff
@@ -0,0 +1,147 @@
+---
+# To contribute improvements to CI/CD templates, please follow the Development guide at:
+# https://docs.gitlab.com/ee/development/cicd/templates.html
+# This specific template is located at:
+# https://gitlab.com/gitlab-org/gitlab/-/blob/master/lib/gitlab/ci/templates/Python.gitlab-ci.yml
+include:
+  - project: nci-gdc/gitlab-templates
+    ref: master
+    file:
+      - templates/global/full.yaml
+      - templates/python/full.yaml
+
+variables:
+ BASE_CONTAINER_VERSION: "1.4.0"
+ PRE_COMMIT_DOCKER_IMAGE: docker.osdc.io/ncigdc/python3.8-builder:2.1.0
+ DOCKER_BUILDKIT: 1
+ PIP_EXTRA_INDEX_URL: https://nexus.osdc.io/repository/pypi-all/simple
+ DOCKER_BUILD_OPTS: "--build-arg PIP_EXTRA_INDEX_URL=https://nexus.osdc.io/repository/pypi-all/simple"
+ DOCKER_PUSH_OPTS: "--all-tags"
+ TWINE_REPOSITORY_URL: "https://nexus.osdc.io/repository/pypi-snapshots/"
+
+
+tox:
+  image: ${BASE_CONTAINER_REGISTRY}/${REPO_PY_VERSION}-builder:${BASE_CONTAINER_VERSION}
+  before_script:
+    - mkdir -p /usr/share/man/man1
+
+.python_versions:
+  parallel:
+    matrix:
+      - REPO_PY_VERSION: ["python36"]
+
+docker_build:
+  allow_failure: true
+  rules:
+    - when: never
```

Comment:
What does this stage do?

Reply:
it suppose to build a docker image for gdcdictionary.

Reply:
What happens if you remove it, not sure why one will need a docker image of the gdcdictionary

Reply:
It will still run if I remove this part. It is defined in the template.